bigstream/distributed_io_utility.py

```diff
@@ -0,0 +1,181 @@
+import glob
```

Comment:
These are some methods from utility.py because I didn't want that file to have any cluster dependencies - other than that there should not be any changes

Reply:
Yeah I remember you did this - it's a great idea. I'll check the rest of the package to make sure anything calling these distributed functions now imports the correct module. Keeping local and distributed code separate as much as possible is really good.